client/stylesheets/_responsive.scss

```diff
@@ -509,6 +538,20 @@
     
 }
 
+@media (max-width: 350px) {
+    .profileCardInfo li:first-child {
+        margin-bottom: 4px !important;
+    }
+
+    .profileCardItem {
+        font-size: 0.8rem !important;
+        margin-bottom: 2px !important;
+    }
+
+    .managerOfferView {
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Selector `managerOfferView` should be written in lowercase with hyphens